prowler/providers/aws/services/redshift/redshift_service.py

```diff
@@ -45,7 +45,8 @@ def _describe_clusters(self, regional_client):
                                 "AllowVersionUpgrade", False
                             ),
                             encrypted=cluster.get("Encrypted", False),
-                            multi_az=cluster.get("MultiAZ", False),
+                            multi_az=str(cluster.get("MultiAZ", False)).lower()
```

Comment:
```suggestion
                            multi_az=cluster.get("MultiAZ", "False)).lower()
```

If it is always a string, we don't need the cast, right? Can we test this, please?

Reply:
I've tested it and it strange, in their docs they say that `it's a string and if true it's enabled` but the sdk parameter is `cluster['MultiAZ'] = 'Enabled'`. So, I'll change that and save that parameter the same way it's returned by Boto3.